internal/microsoft_edge.py

```diff
@@ -70,16 +72,15 @@ def prepare(self, job, task):
             os.makedirs(self.bodies_path)
         try:
             import _winreg
-            keyval = r"SOFTWARE\Classes\Local Settings\Software\Microsoft\Windows\CurrentVersion\AppContainer\Storage\microsoft.microsoftedge_8wekyb3d8bbwe\MicrosoftEdge\Privacy"
-            if not os.path.exists("keyval"):
-                key = _winreg.CreateKey(_winreg.HKEY_CURRENT_USER, keyval)
-            Registrykey = _winreg.OpenKey(_winreg.HKEY_CURRENT_USER, keyval, 0, _winreg.KEY_WRITE)
+            registry_key = _winreg.CreateKeyEx(_winreg.HKEY_CURRENT_USER, self.edge_registry_path, 0, _winreg.KEY_READ)
```

Comment:
since i cant read values which have a KEY_WRITE access, i thought the cleaner approach would be to first set read and than switch to write rights.

Reply:
You can or the access rights with each other: ```_winreg.KEY_READ | _winreg.KEY_WRITE``` so the same key can both read and write.